test/privatekey.js

```diff
@@ -316,7 +316,7 @@ describe('PrivateKey', function() {
 
     it('should parse this uncompressed livenet address correctly', function() {
       var privkey = PrivateKey.fromString(wifLivenetUncompressed);
-      privkey.toString().should.equal(wifLivenetUncompressed);
+      privkey.toString().should.equal("96c132224121b509b7d0a16245e957d9192609c5637c6228311287b1be21627a");
```

Comment:
The test here is checking that the input and output match, perhaps `toWIF` should be used here?